src/app/children/health-checkup/health-check.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ *     This file is part of ndb-core.
+ *
+ *     ndb-core is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     ndb-core is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Entity } from '../../entity/entity';
+import {Child} from '../child';
+
+export class HealthCheck extends Entity {
+
+    static ENTITY_TYPE = 'HealthCheck';
+
+    child: String;      // child_id of the child the health check is made for
+    date: Date;
+    height: number;     // stores the height of a child in cm
```

Comment:
Should we name the attribute explicitly with the unit, e.g. `heightCm` to avoid any misunderstandings? (would be ugly but clear ...)

Can you use JavaDoc style code comments here as well? (`/**` instead of `//`)

Reply:
I don't think that the heightCm thing is a good idea, because it suggests that there is a height variable with a different unit... i would keep it like that for the moment and maybe refactor when we take care of the units properly...